### DISCUSSION TITLE
Ai 配對-日本🇯🇵

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#148] [General]
**新聞內容：**
透過參與者事前先回答大量的問題，系統會根據回答的答案來建構參與者性格，藉此來尋找性格上可能會比較合得來的對象。

最大的好處是排除主觀意識，根據對愛好和價值觀等問題建構的性格分析及系統中的搜索趨勢，即使不符合要求也可以找到「可能對自己有興趣的人」，並提出建議。

內閣辦公室為下一個財政年度的預算要求提供了20億日元， 計劃指定利用人工智慧支援婚姻活動，作為向解決少子化問題的地方政府提供「區域少子化對策重點推進補助金」的專案。

目前有10個縣已經部署了人工智慧系統，崎玉縣在2018年設立邂逅支援中心，成功配對的69組當中，就有33組是AI挑選的結果。

**新聞心得：**
日本大概是在熟悉不過的高齡社會及少子國家，讓AI針對問題分析可能的交往對象，雖然說性格或是興趣可能可以很符合，但是我覺得交往對象還是要看順眼比較重要😂

**新聞來源：**
https://www.ettoday.net/amp/amp_news.php?news_id=1873388&from=amptaglist

**Comment by Timmy110403540** [comment]
AI越來越多工，現在竟然還可以挑情人!

**Comment by pocper1** [comment]
用AI來媒合，我覺得還不錯，可以減少失敗的機會!

**Comment by briankuo123** [comment]
我認為配對好結婚完的當下可能真的是適合的，但是不知道之後他們會不會還覺得合適。

**Comment by wh20502** [comment]
我覺得現在的網路科技發達造成人與人的冷漠，交友也比以往更加具有戒心，沒想到現在要靠科技來拉近人與人的距離了！

**Comment by Shiba010** [comment]
以後連月老都要被AI取代了嗎？！

**Comment by yasklop** [comment]
真的想不到AI能拿來交友配對，真是太厲害了

**Comment by qeeeee39** [comment]
我覺得這只是一個開始，未來機器人月老會越來越令人刮目相看（甚至比月老還靈）

**Comment by ayakasanma** [comment]
感覺很讚，等不及試用了XD

**Comment by Lynn12378** [comment]
配對會有很多考量因素，感情的事靠AI處理，好像沒有辦法那麼符合人性

**Comment by Yunhans** [comment]
Ai愛

**Comment by lx3211** [comment]
用AI來挑選對象啊...連親自看對眼都不需要了，不知道以後感情會不會越來越速食

**Comment by xCk27x** [comment]
讓我想到之前看過的漫畫「戀愛與謊言」
故事背景就是國家透過演算法找出最適合的伴侶
>順便附個連結
https://www.manhuagui.com/comic/14162/

**Comment by brian431** [comment]
感覺兩個人適不適合還是必須聊聊看或出去約會看看，光憑ai分析應該不會比實際接觸還好。雖然速度比較快，但反而可能錯失許多機會。

**Comment by NcuJerryKao** [comment]
可以推薦韓國試用看看 就是那 死亡率高於出生率的國家

**Comment by Ben6uo** [comment]
靠AI分析會可能因為自己選錯的而排除自己喜歡的?

**Comment by todoroki0910** [comment]
以後都要跪求AI給個好對象了

**Comment by skyling0299** [comment]
利用資料庫搜索比對是非常好的點子，或許之後可以變成和AI聊天之後由AI分析比對，推薦比較好的聊天對象!
但是否會造成使用者傾向(覺得AI推薦的一定比較好)而影響實驗準確度也是一個問題

**Comment by limitx0** [comment]
我覺得比起大多數人用外表、學歷、年收那些條件來篩選另一伴
讓AI用雙方的愛好和價值觀來配對的話不僅省時，也許還可以讓配對到的兩人更順利

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Aerofe** [comment]
AI分析可能也只能當作輔助，主要還是要看兩人的互動

**Comment by awehao** [comment]
我看長相怎麼辦

**Comment by skyling0299** [comment]
可是有時候明明知道不是適合的人還是想交往看看，大概是人類的本能? 如果由算法推薦的話會不會少了心動的感覺呢?

**Comment by Raywei0214** [comment]
我覺得找另一伴是緣分的問題，單戀了好久，卻還是只能當備胎emo了

**Comment by titan92601** [comment]
這可以幫現在部會社交的年輕人牽線 或許看促使生育率提高。

**Comment by magicmicheal** [comment]
我認為這適合用於想要步入婚姻的群眾，這應該也是政府投入資金的原因，我想人到了不同階段，應該會對想相處一輩子的人有不同的想法，有人可能認為年輕時，外貌是重點，直到相處後才發現不是這麼一回事

**Comment by bww0401** [comment]
雖然ai挑情人可以增加配對率，但是很多時候想交往的對象，往往跟理想型大不相同，而且人跟人的相處才是進到下段關係的契機，單憑在配對軟體上留下幾句的自我介紹，我認為能達到的效果仍有限

**Comment by number32984** [comment]
AI也太厲害了吧，我們人跟人之間的事情也可以透過AI來解決，那我們人類會不會變得越來越簡單，然後就被AI取代了呢?

**Comment by manipulateed** [comment]
匹配興趣嗜好與外觀等等，其實都不是一對情侶的必須，加值觀與待人處世，才是走得長遠的秘笈，但透過這個配對確實能在話題上面找到第一層篩選，讓人去踏出第一步非常重要。

**Comment by larry40305** [comment]
如果真的變成跟戀愛與謊言類似發展，我覺得會有滿多人喜歡並需要這種配對的。
